util/test/annotate.py

```diff
@@ -45,10 +45,10 @@ def load(path):
   return data
 
 
-def get(data, graph, series, start, end):
+def get(data, graph, series, start, end, hostname=_hostname):
```

Comment:
I agree with both comments, but wasn't sure what to use in place of hostname. machine name, maybe just name?

I also wasn't sure if this script was used anywhere else or if was only called from gengraphs. I was pretty sure only from start_test, but I didn't want to remove _hostname and default arg in case I was wrong.